src/react/components/_dashboard/TasksCard/CompletedTasks/Agreement.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { shortDateIfCurrentYear, parseDate } from 'functions/formatDate';
+
+import styles from './Agreement.module.scss';
+
+import APILink from 'react/components/APILink';
+
+import Task from '../Task';
+import TaskHeader from '../TaskHeader';
+import TaskTitle from '../TaskTitle';
+import CampusSolutionsIcon from '../Icons/CampusSolutionsIcon';
+
+const LinkName = ({ updates }) => {
+  return (
+    <span>
+      {updates ? 'View/Update' : 'View'} <i className="fa fa-arrow-right"></i>
+    </span>
+  );
+};
+
+LinkName.propTypes = {
+  updates: PropTypes.bool,
+};
+
+const Agreement = ({ agreement: task, index }) => {
+  return (
+    <Task index={index} task={task} type="" hasDetail={false}>
+      <TaskHeader task={task}>
+        <CampusSolutionsIcon />
+        <TaskTitle
+          title={task.title}
+          subtitle={`Responded ${task.response} on ${shortDateIfCurrentYear(
+            parseDate(task.responseDate)
+          )}`}
+        />
+
+        {task.url && (
+          <APILink
+            {...task.url}
+            name={<LinkName updates={task.updatesAllowed} />}
+          />
+        )}
+      </TaskHeader>
+    </Task>
+  );
+};
+
+`Responded TKTK }`;
```

Comment:
@developish - What is this? How is this not causing an error?

Reply:
As an empty string on it's own line, it doesn't actually do anything so it doesn't break. Removed.